secure_message/resources/messages.py

```diff
@@ -35,6 +35,11 @@ def post(self):
         if 'msg_id' in post_data:
             raise BadRequest(description="Message can not include msg_id")
         if post_data.get('thread_id'):
```

Comment:
This is validating the message to field ,I think it should be part of  MessageSchema.validate_to instead of inline ?

Reply:
Agreed.  Also moved a bunch of other things in there too that should've always been there